solution/backend/common/functions.py

```diff
@@ -1,72 +1,11 @@
 import boto3
 from django.conf import settings
-from django.contrib.auth.models import Group, Permission
```

Comment:
If we put common functions in the wrong file it causes a circular dependency over the seed data function.  Moved it to its own file to avoid this.